modules/restore-replace-stopped-etcd-member.adoc

```diff
@@ -134,6 +134,13 @@ After you turn off the quorum guard, the cluster might be unreachable for a shor
 etcd cannot tolerate any additional member failure when running with two members. Restarting either remaining member breaks the quorum and causes downtime in your cluster. The quorum guard protects etcd from restarts due to configuration changes that could cause downtime, so it must be disabled to complete this procedure.
 ====
 
+. Delete the affected node by running the following command:
++
+[source,terminal]
+----
+$ oc delete node <node_name>
```

Comment:
Other steps in the procedure use actual values over <variables>. Should we do the same here?
```suggestion
$ oc delete node ip-10-0-131-183.ec2.internal
```

Reply:
I see now that this module in all other supported versions uses <node_name>.  If you make this change, I guess it would need to be CP'd to all including `main` or rebasing this PR against `main`.

Reply:
Thanks @mburke5678 To get this out asap, I will go ahead with this PR as is, then create a new PR to change 4.12+ to your suggestion.